apps/studio/components/interfaces/Auth/Hooks/CreateHookSheet.tsx

```diff
@@ -0,0 +1,352 @@
+                          </div>
+                        }
+                      />
+                      <Radio
+                        key="https"
+                        id="https"
+                        value="https"
+                        label=""
+                        beforeLabel={
+                          <div className="flex items-center space-x-5">
+                            <div className="flex-col space-y-0">
+                              <div className="flex space-x-2">
+                                <p className="text-foreground">HTTPS</p>
+                              </div>
+                              <p className="text-foreground-light">
+                                Used to call any HTTPS endpoint.
+                              </p>
+                            </div>
+                          </div>
+                        }
+                      />
+                    </Radio.Group>
+                  </FormControl_Shadcn_>
+                </FormItemLayout>
+              )}
+            />
+            {values.selectedType === 'postgres' ? (
+              <>
+                <FormField_Shadcn_
+                  key="postgresValues.schema"
+                  control={form.control}
+                  name="postgresValues.schema"
+                  render={({ field }) => (
+                    <FormItemLayout
+                      label="Schema"
+                      className="px-8"
+                      description="Postgres schema where the function is defined."
+                    >
+                      <FormControl_Shadcn_>
+                        <SchemaSelector
+                          size="small"
+                          showError={false}
+                          selectedSchemaName={field.value}
+                          disabled={field.disabled}
+                          onSelectSchema={(name) => {
+                            field.onChange(name)
+                          }}
+                        />
+                      </FormControl_Shadcn_>
+                    </FormItemLayout>
+                  )}
+                />
+                <FormField_Shadcn_
+                  key="postgresValues.functionName"
+                  control={form.control}
+                  name="postgresValues.functionName"
+                  render={({ field }) => (
+                    <FormItemLayout
+                      label="Function name"
+                      className="px-8"
+                      description="Postgres function which will be called by Supabase Auth each time the hook is triggered."
+                    >
+                      <FormControl_Shadcn_>
+                        <FunctionSelector
+                          size="small"
+                          schema={values.postgresValues.schema}
+                          selectedFunctionName={field.value}
+                          onSelectFunction={(name) => field.onChange(name)}
```

Comment:
can we rename these props to just be standard?

Reply:
Fixed.